main.nf

```diff
@@ -759,19 +759,19 @@ process filtlong {
         --keep_percent ${params.longreads_keep_percent} \
         --trim \
         --length_weight ${params.longreads_length_weight} \
-        ${lr} | gzip > ${id}_lr_filtlong.fastq.gz
+        ${lr} | gzip > ${name}_lr_filtlong.fastq.gz
     """
 }
 
 /*
  * Quality check for nanopore reads and Quality/Length Plots
  */
 process nanoplot {
-    tag "$id"
-    publishDir "${params.outdir}/QC_longreads/NanoPlot_${id}", mode: params.publish_dir_mode
+    tag "$name"
+    publishDir "${params.outdir}/QC_longreads/NanoPlot_${name}", mode: params.publish_dir_mode
 
     input:
-    set id, file(lr), type from files_nanoplot_raw.mix(files_nanoplot_filtered)
+    set val(name), file(lr), val(type) from ch_raw_reads_nanoplot.mix(ch_reads_filtered_nanoplot)
```

Comment:
Why not `ch_raw_reads_nanoplot` and `ch_filtered_reads_nanoplot` instead `ch_reads_filtered_nanoplot`